addons/test_mail/tests/test_mail_mail.py

```diff
@@ -1048,9 +1048,9 @@ def send_email(self, message, *args, **kwargs):
                     cr.execute("UPDATE mail_notification SET notification_status='bounce' WHERE id = %s", [notif.id])
             return message['Message-Id']
 
-        self.patch(self.registry['ir.mail_server'], 'send_email', send_email)
-
-        mail.send()
+        with self.mock_mail_gateway():
```

Comment:
Not sure this is correct, there are a lot of added `mock_mail_gateway` that are needed, but are you sure it is not changing the initial purpose of the test? Or are we testing the mock?

Reply:
It is still the same. Now that we disable sending e-mails by default, without a mock, we don't reach the mail server and no special mail server is configured for testing.